components/dev_to/dev_to.app.mjs

```diff
@@ -1,4 +1,28 @@
+import axios from "axios";
```

Comment:
It would be great if you could use axios from `@pipedream/platform`. The only difference is that you need to pass a context as the first param. You can receive this context as a param like in `blogger.app.mjs`. Sorry for not suggesting this before
`@pipedream/platform` will handle some HTTP edge cases and you don't need to access the property `result` as you do using default axios. `@pipedream/platform` will return the `result` for you as the root object